programs/client/Client.cpp

```diff
@@ -2316,6 +2316,7 @@ class Client : public Poco::Util::Application
             ("log-level", po::value<std::string>(), "client log level")
             ("server_logs_file", po::value<std::string>(), "put server logs into specified file")
             ("query-fuzzer-runs", po::value<int>()->default_value(0), "query fuzzer runs")
+            ("history_file", po::value<std::string>(), "path to history file")
```

Comment:
@alexey-milovidov this option is already in use in Client.cpp probably it was removed from config during merge conflict.
```
/// Load command history if present.
if (config().has("history_file"))
    history_file = config().getString("history_file");
else
{
    auto * history_file_from_env = getenv("CLICKHOUSE_HISTORY_FILE");
    if (history_file_from_env)
        history_file = history_file_from_env;
    else if (!home_path.empty())
        history_file = home_path + "/.clickhouse-client-history";
}
```